src/DataStreams/SQLiteBlockInputStream.cpp

```diff
@@ -111,8 +80,33 @@ Block SQLiteBlockInputStream::readImpl()
 void SQLiteBlockInputStream::readSuffix()
 {
     if (compiled_statement)
-    {
         compiled_statement.reset();
+}
+
+void SQLiteBlockInputStream::insertValue(
+    IColumn & column, const ExternalResultDescription::ValueType type, size_t idx)
+{
+    switch (type)
+    {
+        case ValueType::vtUInt8: [[fallthrough]];
+        case ValueType::vtUInt16: [[fallthrough]];
+        case ValueType::vtUInt32: [[fallthrough]];
+        case ValueType::vtUInt64: [[fallthrough]];
+        case ValueType::vtInt8: [[fallthrough]];
+        case ValueType::vtInt16: [[fallthrough]];
+        case ValueType::vtInt32: [[fallthrough]];
+        case ValueType::vtInt64:
+            assert_cast<ColumnUInt64 &>(column).insertValue(sqlite3_column_int64(compiled_statement.get(), idx));
+            break;
```

Comment:
Need to have different `assert_cast<Column* &>(column)` for each case without `fallthrough`.
Also `sqlite3_column_int64` - what about uint64? It will fail with this code. There should be more type options in the library.